src/components/Modal/Dialog.tsx

```diff
@@ -1,94 +1,72 @@
 import * as React from 'react';
 import { themr, ThemedComponentClass } from 'react-css-themr';
+import { classNames } from '@shopify/react-utilities/styles';
+
 import { MODAL } from '../ThemeIdentifiers';
 import Heading from '../Heading';
 import Button from '../Button';
-import helpers from './helpers';
-import * as baseTheme from './Modal.scss';
-
-export enum SizeType {
-  Small = 300,
-  Medium = 600,
-  Large = 800,
-}
 
-export type Size = 'Small' | 'Medium' | 'Large' | number;
+import * as baseTheme from './Modal.scss';
 
-export interface SizeStyle {
-  maxWidth: string;
-  width: string;
-}
+export type Width = 'small' | 'medium' | 'large' | string;
```

Comment:
@dheeraja00 It seems we're defining this propertyin multiple places.  let's reuse.